src/GRA.Controllers/MissionControl/ReportingController.cs

```diff
@@ -81,10 +81,12 @@ public IActionResult Index()
             var schoolList = await _schoolService.GetSchoolsAsync(schoolDistrictList.FirstOrDefault()?.Id);
             var groupInfoList = await _userService.GetGroupInfosAsync();
             var vendorCodeTypeList = await _vendorCodeService.GetTypeAllAsync();
+            var site = await GetCurrentSiteAsync();
 
             return View($"{viewName}Criteria", new ReportCriteriaViewModel
             {
                 ReportId = id,
+                ProgramStartDate = site.ProgramStarts ?? DateTime.Parse("2018/01/01"),
```

Comment:
This should probably be `new DateTime(2018, 01, 01)` because `DateTime.Parse` is probably dependent on culture settings.